homeassistant/components/kulersky/light.py

```diff
@@ -158,6 +158,11 @@ async def async_update(self) -> None:
         if not brightness:
             self._attr_rgbw_color = (0, 0, 0, 0)
         else:
-            rgbw_normalized = tuple(round(x * 255 / brightness) for x in rgbw)
-            self._attr_rgbw_color = rgbw_normalized  # type:ignore[assignment]
+            rgbw_normalized = [round(x * 255 / brightness) for x in rgbw]
+            self._attr_rgbw_color = (
+                rgbw_normalized[0],
+                rgbw_normalized[1],
+                rgbw_normalized[2],
+                rgbw_normalized[3],
+            )
```

Comment:
This change makes it possible to drop the `type: ignore` statement.
I couldn't find a way to make it prettier.